libs/features/spectrum2-styles/build.mjs

```diff
@@ -0,0 +1,84 @@
+import fs from 'fs';
+import path from 'path';
+import postcss from 'postcss';
+
+const spectrumCSSPath = path.resolve('node_modules/@spectrum-css/tokens/dist/index.css');
+const miloCSSPath = path.resolve('../../deps/spectrum2-styles.css');
```

Comment:
I see that we are reading the CSS too - have a separate question about that.

Depending on your answer, I'd suggest to rename this:
```suggestion
const outputPath = path.resolve('../../deps/spectrum2-styles.css');
```

Reply:
Tried to clarify this a bit below, but we do read the miloCSSPath to define the custom properties we'll need from spectrum css.